translator/translate/otel/exporter/awscloudwatch/translator.go

```diff
@@ -164,6 +171,14 @@ func getDropOriginalMetrics(conf *confmap.Conf) map[string]bool {
 		*/
 		if dropMetrics := common.GetArray[any](conf, dropOriginalCfgKey); dropMetrics != nil {
 			for _, dropMetric := range dropMetrics {
+				if _, in := toDropMap[category]; in {
```

Comment:
nit: I think ["never-nesting"](https://www.martin-paucot.fr/blog/stop-nesting-your-code-4ng8) makes code more readable and easier to debug:

Reply:
Will adjust.